processing/src/main/java/io/druid/jackson/DefaultObjectMapper.java

```diff
@@ -53,7 +53,7 @@ public DefaultObjectMapper(JsonFactory factory)
 
     configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
     configure(MapperFeature.AUTO_DETECT_GETTERS, false);
-//    configure(MapperFeature.AUTO_DETECT_CREATORS, false); https://github.com/FasterXML/jackson-databind/issues/170
+    configure(MapperFeature.AUTO_DETECT_CREATORS, false);
```

Comment:
This change has some risk, it's possible that it breaks deserialization for something we don't know about because this code has been commented out for a long time. I think it is "better" to have `AUTO_DETECT_CREATORS` disabled but this should be a separate patch, since its risk is not adequately captured by the title "Add Checkstyle check to prohibit IntelliJ-style commented code lines". We would also need to be on the lookout for regressions caused by the change.

Maybe in this patch, change the line to not match the regex (use `//` with one space not multiple) and do a follow up for the `AUTO_DETECT_CREATORS` thing if you want to change that behavior too.